docs/gettingstarted/buildingtest.rst

```diff
@@ -138,6 +138,13 @@ tags by running ``buildtest buildspec find --tags``.
 .. Note:: The ``--tags`` is used for discovering buildspec file and not filtering tests
    by tag.
 
+You can also build by multiple tags by providing tagnames as comma seperated list. In the
```

Comment:
spelling error `seperated` -> `separated`.

Also you can change this sentence `You can also build by multiple tags by providing ...` --> `You can specify multiple tag names as a comma separated list.`.